cfme/tests/containers/test_containers_default_views.py

```diff
@@ -23,11 +31,12 @@
     'Containers Providers': 'containers_providers',
     'Projects': 'containers_projects',
     'Routes': 'containers_routes',
-    # 'Nodes': 'containers_nodes',
```

Comment:
We can now do the mapping again like this
```python
mapping = {
    'Containers Providers': ContainersProvider,
    'Projects': Project
    ...
}

So that below..... it becomes ...